frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ManagedClusters.tsx

```diff
@@ -837,11 +842,12 @@ export function useClusterCreatedDateColumn(): IAcmTableColumn<Cluster> {
       )
     },
     search: 'creationDate',
-    cell: (cluster) => {
-      const dateTimeCell = getDateTimeCell(
-        cluster.creationTimestamp ? new Date(cluster.creationTimestamp).toString() : '-'
-      )
-      return dateTimeCell.title === 'Invalid Date' ? '-' : dateTimeCell.title
-    },
+    cell: (cluster) => (
+      <span style={{ whiteSpace: 'nowrap' }}>
+        <TextContent>
+          <Text component={TextVariants.small}>{getDateTime(cluster)}</Text>
+        </TextContent>
+      </span>
+    ),
```

Comment:
```suggestion
    cellTransforms: [nowrap],
    cell: (cluster) => getDateTime(cluster),
```
Add import `nowrap` from `@patternfly/react-table`

Reply:
With the components you added, the text appears in a shade of grey. Using the cellTransform achieves the same in less code without changing the text color.

Reply:
made the change -- wow--used how another field had nowrap in this same component--apparently James hadn't heard of pf nowrap transform